migrations/20200917163300-create-events.js

```diff
@@ -0,0 +1,22 @@
+exports.up = (db, callback) => db.createTable('event', {
+  id: { type: 'int', primaryKey: true, autoIncrement: true },
+  type: 'string',
+  label: 'string',
+  model: 'string',
+  modelId: 'int',
+  createdAt: { type: 'date', notNull: true },
+  body: { type: 'jsonb', defaultValue: '{}' },
+})
+  .then(() => Promise.all([
+    db.runSql('CREATE INDEX IF NOT EXISTS "event_type_idx" ON event USING btree ("type");'),
+    db.runSql('CREATE INDEX IF NOT EXISTS "label" ON event USING btree ("label");'),
```

Comment:
I'm not sure that we'll ever query for "all errors" or "all audits". I'm guessing it will typically be something including `createdAt` as well. Given that write performance is probably more important that read performance since we want to be able to create lots of events and not impact the running system, I'd lean toward creating fewer indexes up front and adding them once we see how we are querying the data.